apps/vendor-portal/src/modules/offers/pages/offers-details.vue

```diff
@@ -675,17 +672,55 @@ async function onImageUpload(files: FileList) {
   }
 
   files = null;
+};
+
+const onGalleryItemEdit = (item: Image) => {
+  emit("page:open", {
+    component: AssetsDetails,
+    componentOptions: {
+      editableAsset: item,
+      images: offerDetails.images,
+      sortHandler: sortImage,
+    },
+  });
+};
+
+function sortImage(remove = false, localImage: IImage) {
```

Comment:
I think this logic must be moved out of here into composable level.

Reply:
done